packages/cozy-harvest-lib/src/components/TriggerManager.jsx

```diff
@@ -97,16 +97,35 @@ export class DumbTriggerManager extends Component {
    * OAuth Form success handler. OAuthForm retrieves an account id created by the
    * cozy stack
    * @param  {string}  accountId
+   * @param  {integer} bankId
    */
-  async handleOAuthAccountId(accountId) {
+  async handleOAuthAccountId(accountId, bankId) {
     const { client, flow, konnector, t } = this.props
-    const oAuthAccount = await fetchAccount(client, accountId)
-    flow.ensureTriggerAndLaunch(client, {
-      account: oAuthAccount,
-      konnector: konnector,
-      trigger: flow.trigger,
-      t: t
-    })
+    let oAuthAccount = await fetchAccount(client, accountId)
+
+    const konnectorPolicy = findKonnectorPolicy(konnector)
+
+    let foundWebauthConnection = false
+    if (konnectorPolicy.handleWebauthAccount) {
+      foundWebauthConnection = await konnectorPolicy.handleWebauthAccount({
```

Comment:
Webauth is very specific to BI. I think TriggerManager should not know about it. Could we find a generic name for the konnector policy method ?